docs/pages/components/modal.mdx

```diff
@@ -123,7 +123,7 @@ function() {
       </Modal.Trigger>
       <Modal store={modal} ariaLabel="example">
         <Modal.Content store={modal}>
-          <Modal.Header title={title} subtitle={subtitle} icon={icon}/>
+          <Modal.Header title={<div>hello</div>} subtitle={subtitle} icon={icon}/>
```

Comment:
Remove this example :) we need to avoid maximum the use of other than string ^^

Reply:
oops, just forgot to remove when i test it 🙈